src/Azure.Functions.Cli/ExtensionBundle/ExtensionBundleHelper.cs

```diff
@@ -42,5 +44,24 @@ public static string GetBundleDownloadPath(string bundleId)
         {
             return Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), Constants.UserCoreToolsDirectory, "Functions", ScriptConstants.ExtensionBundleDirectory, bundleId);
         }
+
+        public static async Task GetExtensionBundle()
+        {
+            var extensionBundleManager = GetExtensionBundleManager();
+            try
+            {
+                using (var httpClient = new HttpClient())
+                {
+                    httpClient.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
Timeout is now 5 mins instead of [100 seconds](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=net-6.0#remarks).